CHANGELOG.md

```diff
@@ -7,7 +7,9 @@ We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 -
 
 ### Changed
--
+- **all components:** Corrected the variables
+  - Updated all the variables to the correct names, locations, and organization
+  - Updated all the docs to reflect the correct variables
```

Comment:
Oh right. I'm also not sure how the `all components` here will shake out with the other PRs we've just merged if they're all released as one. @contolini can you confirm this was the issue with the v4 release or was it something else?

Either way, I also realized I forgot a `[MINOR]` in here